public/components/notebooks/components/__tests__/notebook.test.tsx

```diff
@@ -3,8 +3,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { fireEvent, render, waitFor } from '@testing-library/react';
+import { fireEvent, render, waitFor, act } from '@testing-library/react';
```

Comment:
React-test `act()` introduced in a couple of tests to fix async operations and correct waiting for render-cycle between do and expect.